src/Plotly.NET.Interactive.Extension/Plotly.NET.Interactive.Extension.fsproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
As this is a library, can we target netstandard?

Reply:
You are totally right! For some reason I thought, that the .NET Interactive nuget dependency was still using `netcoreapp` buth either they switched to `netstandard2.1` or I just mixed it up (very likely 🙈). I'll change it to `netstandard2.1`.